src/types.ts

```diff
@@ -6,6 +6,7 @@ export type DynamicControlSection = {
     documentation?: string
     collapsible?: boolean
     collapsed?: boolean
+    hiddenByDefault?: boolean
```

Comment:
Why did this need adding?

Reply:
I think this was auto-generated. I left it in because I thought it might be important. I'll remove it now.